server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/VacationIntegrationTest.java

```diff
@@ -376,7 +376,7 @@ private void assertOneMessageReceived(AccessToken recipientToken, String mailbox
             .body(SECOND_ARGUMENTS + ".list[0].to.email[0]", equalTo(expectedTo));
     }
 
-    private boolean assertNotYetImplemented(AccessToken recipientToken, String mailboxId) {
+    private boolean assertOneHTMLMessageReceived(AccessToken recipientToken, String mailboxId, String expectedFrom, String expectedTo) {
```

Comment:
Because it is not supported yet.... There is nothing to recognise an attachement with text/html or text/plain and use them as textBody or textHtml.

I can do this on Message::fromMailboxMessage but this is definitly an other ticket. I am sure you don't want to wait ~4 days to fix the build.